conjure-go-server/httpserver/handlers.go

```diff
@@ -123,3 +124,26 @@ func ErrHandler(ctx context.Context, statusCode int, err error) {
 		svc1log.Stacktrace(err),
 	)
 }
+
+func getSerializableCause(err error) error {
+	if conjureErr := errors.GetConjureError(err); conjureErr != nil {
+		return conjureErr
+	}
+	if marshaler := getJSONMarshaler(err); marshaler != nil {
```

Comment:
@bmoylan is this approach acceptable for non-conjure errors that implement `json.Marshaler`? Or are there reasons to use `werror.RootCause()` instead and only look at the base error?

Reply:
Talked offline and agreed this is probably the right approach.